README.md

```diff
@@ -3,3 +3,76 @@ Client tools for using CADC services
 
 Build status:
 <a href="https://travis-ci.org/opencadc/cadctools"><img src="https://travis-ci.org/opencadc/cadctools.svg?branch=master" /></a>
+
+# Developers Guide
+
+Requires Python version: 2.7, 3.3, 3.4, 3.5.
+
+Requires pip.
+
+
+## Installing Packages
+
+Run 
+```
+cd cadcutils && pip install -r ./dev_requirements.txt
+cd cadcdata && pip install -r ./dev_requirements.txt
+pip install cadcutils/
+pip install cadcdata/
```

Comment:
These two lines are redundant as these installs already occur in dev_requirements.txt